src/test/kotlin/io/github/orangain/prettyjsonlog/logentry/TimestampTest.kt

```diff
@@ -36,4 +38,18 @@ class TimestampTest : TestCase() {
             assertEquals("Timestamp.fromEpoch($input)", Timestamp.Parsed(Instant.parse(expected)), actual)
         }
     }
+
+    fun testExtractTimestampWithSecondsAndNanos() {
+        val testCases = listOf(
+            Pair(Instant.ofEpochSecond(1000, 500), """{"timestampSeconds": 1000, "timestampNanos": 500}"""),
+            Pair(Instant.ofEpochSecond(0, 0), """{"timestampSeconds": 0, "timestampNanos": 0}"""),
+            Pair(Instant.ofEpochSecond(1000000000, 123456789), """{"timestampSeconds": 1000000000, "timestampNanos": 123456789}""")
+        )
+
+        testCases.forEach { (expectedInstant, jsonString) ->
+            val jsonNode = objectMapper.readTree(jsonString) as ObjectNode
```

Comment:
Unfortunately, compilation of the test has failed with the following error:

```
> Task :compileTestKotlin FAILED
e: file:///home/runner/work/pretty-json-log-plugin/pretty-json-log-plugin/src/test/kotlin/io/github/orangain/prettyjsonlog/logentry/TimestampTest.kt:50:28 Unresolved reference 'objectMapper'.
```

https://github.com/orangain/pretty-json-log-plugin/actions/runs/11363964014/job/31614617283?pr=83

Reply:
Apologies my build env. is stuffed, hope all ok now.